src/modules/profile/components/ProfileList/ProfileListItem.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+
+import { StyledItemWrapper, StyledLabelWrapper, StyledInputWrapper } from './ProfileList.styled';
+import { RegularText } from "../../../../core/components/textComponents/TextComponents";
```

Comment:
> ../../../../

this starts to be annoying :)
maybe read the article about absolute/relative path?

Reply:
Yeah. But it was implemented in previous pull request, which is still unapproved yet. :)